### DISCUSSION TITLE
Ultratemp Heat Setpoint not working on MQTT

### DISCUSSION THREAD

**Discussion by Tegguy** [tagyoureit/nodejs-poolController#599] [App and O/S Config Questions] [answered]
I have recently updated to the latest NJSPC code and I noticed my home assistant integration stopped showing the proper heat setpoint. The cool setpoint is still working but the "heatsetpoint" now shows up under "setpoint" rather than "heatsetpoint". I don't know if this causes any issues at this point but was curious if there is a way to get it to show up under the correct item so I don't have to change the home assistant code. This is true for both the pool body and the spa body.

![image](https://user-images.githubusercontent.com/83373401/189487661-2396fe70-8b2a-457b-b213-3d25ddd0a10c.png)

**Comment by rstrouse** [comment, marked as answer]
Was your binding local?   I couldn't find any point in time where the default bindings were setting a heatSetpoint.  The setPoint was always being set as the heating setpoint.  No matter though if you pull njsPC it will replicate the setPoint to heatSetpoint.

**Reply by Tegguy** [reply]
Interesting... I don't remember setting up a local binding but maybe I did since it's been a while since I got it working. I'll pull the latest njsPC when I get home.

Thank You

**Reply by Tegguy** [reply]
This got it working thank you!
